production/loki-mixin/README.md

```diff
@@ -0,0 +1,44 @@
+# loki-mixin
+
+loki-mixin is a jsonnet library containing a set of loki monitoring dashboards, alerts and rules used on our production setup. 
+
+## Dashboards
+
+To test the dashboards against a local grafana & loki setup perform the following steps.
+
+### Pre-requisites
+
+* Grizzly is a tool for managing jsonnet dashboards in Grafana: https://github.com/grafana/grizzly.
+To install it follow the instructions at: https://grafana.github.io/grizzly/installation/.
+
```

Comment:
Should we also add `jb` tool as pre-requisites?

Reply:
Done in: https://github.com/grafana/loki/pull/6083/commits/379db167b08310e4af98ac9b8ef4612424366d6e